charmcraft.yaml

```diff
@@ -4,18 +4,9 @@ type: charm
 bases:
   - build-on:
     - name: "ubuntu"
-      channel: "20.04"
+      channel: "22.04"
       architectures: ["amd64"]
     run-on:
-    - name: "ubuntu"
```

Comment:
whats the reason for removing focal from run on?

Reply:
great question -- and the answer is [i don't know yet](https://chat.charmhub.io/charmhub/pl/bzahkmytzjf38gfyiw3znbdefr)
i'm trying to determine how necessary this series definition for kubernetes charms

From my understanding, the charm runs in a jujud-operator based on the version of the juju model it's running in on kubernetes.  I *THINK* the series definition within a jujud-operator container is fixed -- so the charm doesn't really need to specify the series because it won't matter.  it's running in a container and that's that...